docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe/hosted-payment-page.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Project guidelines for Stripe - Hosted Payment Page
+description: Learn how to implement Stripe using ACP
+last_updated: Nov 8, 2024
+template: howto-guide-template
+related:
+   - title: Stripe
+     link: docs/pbc/all/payment-service-provider/page.version/base-shop/third-party-integrations/stripe/stripe.html
+redirect_from:
+   - /docs/pbc/all/payment-service-provider/202311.0/third-party-integrations/stripe/install-stripe.html
+   - /docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/install-stripe.html
+   - /docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/integrate-stripe.html
+
+---
+
+### Implementing Stripe checkout as a hosted payment page
+
+If you have rewritten `@CheckoutPage/views/payment/payment.twig` on the project level, do the following:
```

Comment:
How is this related to a hosted payment page?

Reply:
@matweew do you remember why this code was needed exactly? It is a copy from the original file project-guidelines-for-stripe.md which was added by you.

Reply:
These steps are needed **only** if the project is not fresh or the Checkout Payment page on Yves is changed.
 
But in general, this code is required to work correctly with any ACP payment method including ones that use a hosted payment page (redirect) approach).

Reply:
so the header could be not very correct 
`Implementing Stripe checkout as a hosted payment page`

and better to change to something like `Prerequisites for Yves Checkout before using Stripe as hosted payment page`

Reply:
@matweew Do you mean that this whole document is a prerequisite?

Reply:
integration guide, what should be done or checked to use Stripe app in hosted payment page mode